### PR TITLE
Use min relay value constant in QT

### PR DESCRIPTION
**Pull request by instagibbs**

### CODE REVIEW

**Comment by luke-jr** [comment]
I guess it's a strict improvement, but shouldn't it be using the user-configured value??

**Comment by instagibbs** [comment]
@luke-jr this value is the one that "rejects" too-low values, aka something that simply cannot propogate. 

(although I guess this can change too depending on current mempool minfee, eh?)

File: src/qt/sendcoinsdialog.cpp [hunk 1]
```diff
@@ -175,7 +176,7 @@ void SendCoinsDialog::setModel(WalletModel *_model)
         connect(ui->customFee, &BitcoinAmountField::valueChanged, this, &SendCoinsDialog::coinControlUpdateLabels);
         connect(ui->optInRBF, &QCheckBox::stateChanged, this, &SendCoinsDialog::updateSmartFeeLabel);
         connect(ui->optInRBF, &QCheckBox::stateChanged, this, &SendCoinsDialog::coinControlUpdateLabels);
-        CAmount requiredFee = model->wallet().getRequiredFee(1000);
+        CAmount requiredFee = model->wallet().getRequiredFee(DEFAULT_MIN_RELAY_TX_FEE);
```
Comment (MarcoFalke) [inline src/qt/sendcoinsdialog.cpp]: Huh? The units don't match up here. 1000 is of unit kB and the constant you put in here is a feerate.

**Review by maflcko — CHANGES_REQUESTED** [review]
NACK, not sure what this is trying to solve

**Comment by instagibbs** [comment]
hmm I thought I tested this locally but this is obviously wrong, closing

**Comment by luke-jr** [comment]
> aka something that simply cannot propogate

Literally anything CAN propagate.

Network policies are not constants, and should assume to vary.

**Comment by instagibbs** [comment]
Not sure why you're necroing this PR.

**Pull request closed without merging**